.travis.yml

```diff
@@ -23,8 +22,7 @@ cache:
 
 install:
   # Install Ansible
-  - pip install "ansible~=$ANSIBLE_VERSION"
-
+  - pip install "ansible~=$ANSIBLE_VERSION.0"
```

Comment:
Are you sure about this one?

Because it seems to install the first released subversion of each ones, which can be an issue because it doesn't include bugfixes.
I also use it here for example, and the last build passed 4 days ago: https://github.com/aruhier/ansible-role-snmpd/blob/master/.travis.yml

Reply:
AFAIK if you use `pip install "ansible~=2.5"` it'll install the latest of the 2.x branch.
If you use `pip install "ansible~=2.5.0"` it'll install the latest of the 2.5.x branch.

I might be wrong, but as I understand it, all of your tests seems to use Ansible 2.10.

Here's a test on my laptop:
```
pip3 install "ansible~=2.5"
Collecting ansible~=2.5
  Downloading https://files.pythonhosted.org/packages/a0/53/881aa200bd3edf2e25e865629443791a02b3f2c4a8d8eef05f8bab0bd8c5/ansible-2.10.6.tar.gz (29.6MB)
```

```
 pip3 install "ansible~=2.5.0"        1 ↵
Collecting ansible~=2.5.0
  Using cached https://files.pythonhosted.org/packages/b3/b4/843ad4e43bb36b32ce2ed9a2fd7626915cfb16b05c19c60e9b2bb78898ef/ansible-2.5.15.tar.gz
```

Reply:
Wow, that's indeed something I really didn't expect. Yes, you're definitely right, thanks for the explanation!